packages/benchmarks/.env

```
@@ -1 +1 @@
-DATABASE_URL=postgres://postgres:@localhost:5432/orchid-orm-benchmarks
+DATABASE_URL=postgres://postgres:postgres@localhost:5432/orchid-orm-benchmarks
```

```suggestion
DATABASE_URL=postgres://postgres:password@localhost:5432/orchid-orm-benchmarks
```

Let it be `password`?

---

I don't mind any none empty string as long as it avoids Error: SASL: SCRAM-SERVER-FIRST-MESSAGE